src/main/java/com/vistatec/ocelot/segment/view/SegmentTableModel.java

```diff
@@ -190,7 +190,7 @@ public Object getValueAt(int row, int col) {
 
     @Override
     public boolean isCellEditable(int row, int col) {
-        return col == getSegmentTargetColumnIndex();
+        return col == getSegmentTargetColumnIndex() || col == getSegmentSourceColumnIndex();
```

Comment:
This change causes some weird side effects for me on the mac -- double-clicking a source cell now inverts the text, but doesn't cause any functional change.